tests/test_photo.py

```diff
@@ -83,7 +83,8 @@ def test_creation(self, thumb, photo):
     def test_expected_values(self, photo, thumb):
         assert photo.width == self.width
         assert photo.height == self.height
-        assert photo.file_size == self.file_size
+        # For some reason something changed at some point, that's why we allow two values here …
+        assert photo.file_size in [self.file_size, 27662]
```

Comment:
wont that break the general comparison experience between two photo object? or rather: Why are there two values?

Reply:
because in recent tests suits for some reason in *some* (not all) os-py version combinatitions the second file size is generated …

Reply:
consistent?

Reply:
yes, I've only ever seen 27662 as resulting file size in the failing tests. But, well, similar as for the other tests, it's not really our job to check the size and file size of the resulting photo, IG, we only want to make sure that a photo __is__ sent. If/How it's compressed is TGs decision. So we could also just drop those tests, if you're more comfortable with that

Reply:
I think I would be, yeah. Better then having to add a random number and hope telegram doesn't change how they compress

Reply:
All right. Only dropped the file size test for now and kept dimensions. The comment should be clear enough in case that fails in the future, too.

Reply:
sure